play/src/pages/PhaseResults.tsx

```diff
@@ -3,13 +3,9 @@ import {
   GameSessionState,
   IGameSession,
```

Comment:
`PhaseResults` is impacted by hoisting `answerChoices` to `gameSessionContainer` and `getSelectedAnswer` to `ModelHelper`